test/sequential/test-child-process-emfile.js

```diff
@@ -1,30 +1,46 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var spawn = require('child_process').spawn;
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const child_process = require('child_process');
+const fs = require('fs');
 
 if (common.isWindows) {
   console.log('1..0 # Skipped: no RLIMIT_NOFILE on Windows');
   return;
 }
 
-var openFds = [];
+const ulimit = Number(child_process.execSync('ulimit -n'));
```

Comment:
I'd use `ulimit -Hn` for querying, `ulimit -n num` is fine for setting it because it sets both the hard and soft limit.  Just querying for `ulimit -n` would be wrong in the (admittedly unlikely) case that the soft limit is <= 64 but the hard limit is > 64.